modules/cnf-performing-end-to-end-tests-running-in-single-node-cluster.adoc

```diff
@@ -24,21 +19,24 @@ When executing `podman` commands as a non-root or non-privileged user, mounting
 
 * You have logged in as a user with `cluster-admin` privileges.
 
+* You have applied a cluster performance profile by using Node Tuning Operator.
+
 .Procedure
 
 * To run the latency tests on a {sno} cluster, run the following command:
 +
 [source,terminal,subs="attributes+"]
 ----
 $ podman run -v $(pwd)/:/kubeconfig:Z -e KUBECONFIG=/kubeconfig/kubeconfig \
--e DISCOVERY_MODE=true -e FEATURES=performance -e ROLE_WORKER_CNF=master \
-registry.redhat.io/openshift4/cnf-tests-rhel8:v{product-version} \
-/usr/bin/test-run.sh -ginkgo.focus="\[performance\]\ Latency\ Test"
+-e LATENCY_TEST_RUNTIME=<time in seconds> registry.redhat.io/openshift4/cnf-tests-rhel8:v{product-version} \
```

Comment:
For consistency with other work in this PR and also to avoid improper syntax highlighting in the output (it mistakes `time` for a command), make it <time_in_seconds>